src/helpers/githubApi.ts

```diff
@@ -0,0 +1,34 @@
+let githubToken = '';
+
+export const saveToken = (token: string) => {
+  githubToken = token;
+  localStorage.setItem('github_token', token);
+};
+
+export const getToken = () => {
+  if (!githubToken) {
+    githubToken = localStorage.getItem('github_token') || '';
+  }
+  return githubToken;
+};
+
+export const githubRequest = async (endpoint: string, options: RequestInit = {}) => {
+  const token = getToken();
+  if (!token) {
+    throw new Error('GitHub Token 未设置');
+  }
+
+  const response = await fetch(`https://api.github.com${endpoint}`, {
+    ...options,
+    headers: {
+      ...options.headers,
+      Authorization: `Bearer ${token}`,
+    },
```

Comment:
This is causing the bug. options has the same thing as headers, they are duplicate. I think we can just keep `...options` and remove the rest.